heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/HttpJsonClient.java

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.twitter.heron.common.network;
+package com.twitter.heron.scheduler.kubernetes;
```

Comment:
I would suggest not to move the file - this is going to be used in DC/OS and ECS later. Hence the original place was more apt and we take a dependency on jackson and we shade jackson where it is being used (like instance).